portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,11 @@
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA-hZxZK2Q5trCATeE4yc31-oZQvDrhTCM"></script>
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="loginBasedDisplay(); createMap()">
```

Comment:
you can move this event handler into the javascript file. somethig like
window.addEventListener('load', function() {
  loginBasedDisplay();
  createMap();
});